### DISCUSSION TITLE
How can i merge mesh or geometry after using GLTFLoader?

### DISCUSSION THREAD

**Discussion by RyugaRyuzaki** [zeux/meshoptimizer#669] [Q&A]
Hi, i need someone help me how can i merge mesh or geometry after this:
```
import { MeshoptDecoder } from './meshopt_decoder.module.js';
var loader = new GLTFLoader();
loader.setMeshoptDecoder(MeshoptDecoder);
loader.load('pirate.glb', function (gltf) { scene.add(gltf.scene); });
```
Here is output 
![image](https://github.com/zeux/meshoptimizer/assets/89787521/889aba8a-19fe-4a91-9c18-f1cab7788bda)
Thank you !

**Comment by donmccurdy** [comment]
It appears that you're asking mainly about three.js APIs - the three.js forums really would be the best place for a quicker response! I would recommend having a look at [THREE.BufferGeometryUtils](https://threejs.org/docs/?q=buffergeomet#examples/en/utils/BufferGeometryUtils.mergeGeometries) and follow up on the forums if you're still having trouble.
